### DISCUSSION TITLE
Quest: Noexit=yes

### DISCUSSION THREAD

**Discussion by mpikzink** [streetcomplete/StreetComplete#3702] [Ideas]
I still find paths in the woods or in some other places that have not been mapped to the end. I think it would make sense to quest open ended paths that don't have a noexit=yes tag. Either they are bugs from other mapper because the adjacent path is not connected, the noexit=yes tag would not be set at the end of the path or the path still needs to be completed. What so you experts think?

**Comment by matkoniecz** [comment]
See https://github.com/streetcomplete/StreetComplete/blob/master/.github/ISSUE_TEMPLATE/quest-suggestion.md

The problem is: what should be done in cases where answer is "path continues"?

It would likely produce absurd amount of notes that even with photo require on-place survey by mapper using a full editor.

**Comment by andrewharvey** [comment]
> The problem is: what should be done in cases where answer is "path continues"?

Tag [`fixme=continues`](https://taginfo.openstreetmap.org/tags/fixme=continues)

**Reply by Helium314** [reply]
I think it should rather be `fixme=continue` (without __s__), which is the suggested couterpart to `noexit=yes` https://wiki.openstreetmap.org/wiki/Tag:noexit=no

**Reply by matkoniecz** [reply]
Hmmmmmm.

It may actually work.

Yes, it would create many new `fixme` but it is not actually obstructive and I often manually created such notes as an initial survey step

The main question: is it worth effort of SC mapper? I think that answer is also yes. Yes, it would apply some in cases where it is clear from aerial view.

May it would be actually a good idea to at least create an issue for that ( https://github.com/streetcomplete/StreetComplete/issues/new?assignees=&labels=&template=quest-suggestion.md ) and review criteria once more.

**Reply by andrewharvey** [reply]
> I think it should rather be `fixme=continue` (without **s**), which is the suggested couterpart to `noexit=yes` https://wiki.openstreetmap.org/wiki/Tag:noexit=no

looks like both are used, with one much more common

this path "continues", vs "continue" this path

**Reply by andrewharvey** [reply]
> The main question: is it worth effort of SC mapper? I think that answer is also yes. Yes, it would apply some in cases where it is clear from aerial view.

I think so, data consumers can render a path endpoint which is know to continue differently to one known to not continue.

**Reply by matkoniecz** [reply]
This one remain purely speculative use, I would rather consider rather use for other mappers.
